hack/release.toml

```diff
@@ -79,6 +79,23 @@ config:
         - content: MONITOR ${upsmonHost} 1 remote pass password
           mountPath: /usr/local/etc/nut/upsmon.conf
 ```
+"""
+
+ [notes.extensions-environment]
```

Comment:
it's a nit, but it might be easier to merge these two notes in a single item, as they both talk about same are - configuring extension services